docs/release_notes/2024/april2024.md

```diff
@@ -9,6 +9,22 @@ tags:
   - march
 ---
 
+## 3.3.5.\* (April 26,2024)
+
+- Prophecy Python libs version: 1.8.15
+- Prophecy Scala libs version: 7.1.93
+
+### Features {#Features335}
+
+#### Parametrized Gems in Spark
+
+Many enterprises use generic Pipelines that act like frameworks. These Pipelines have the same basic transformation rules but work with files that have different schemas. They generally have different output columns, computed using different expressions.
+To help with this, Prophecy now lets you completely parameterize your transformation gems. This means the logic for all the expressions along with output columns can come from a Pipeline config. So, you can use one Pipeline as a framework and run it with different setups. Learn more about this feature [here](../../low-code-spark/expression-builder.md).
+
+#### Copilot Fix It for Spark and SQL expressions
+
+Also, in our last update, we'd enabled Copilot to suggest expressions automatically based on output column names. With this release, we've added a feature called Copilot Fix It. It can fix any errors in your expression logic with just a click of the "Fix with AI" button. It can correct things like syntax mistakes, wrong function calls, or typos. We'll have a detailed document on this feature soon.
```

Comment:
If multiple errors are present within a gem, users can navigate to the graph view, select the triple dots associated with the gem, and correct all errors simultaneously.